precompiles/staking/method_distribute.go

```diff
@@ -0,0 +1,116 @@
+package staking
+
+import (
+	"fmt"
+	"math/big"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/vm"
+
+	ptypes "github.com/zeta-chain/node/precompiles/types"
+	fungibletypes "github.com/zeta-chain/node/x/fungible/types"
+)
+
+var (
+	zrc20lockerAddress = common.HexToAddress("0x0000000000000000000000000000000000000067")
```

Comment:
How do we get this address ? some comments would be helpful

Reply:
This address is the bank precompile address, is it the address under which bank and distribute lock the ZRC20 tokens. Adding an explanatory comment soon.

Reply:
We have a variable for the bank address? Why not reusing it?

Reply:
Modified in https://github.com/zeta-chain/node/commit/06d0c5d5a8a2d7ce54f84a666c7bef7784963ccf